pkg/api/types.go

```diff
@@ -836,9 +942,13 @@ type ResourceName string
 
 const (
 	// CPU, in cores. (500m = .5 cores)
-	ResourceCPU ResourceName = "cpu"
+	ResourceCPU			ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
-	ResourceMemory ResourceName = "memory"
+	ResourceMemory 		ResourceName = "memory"
+	// volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 2014 * 2014)
```

Comment:
Size is very vague, but I haven't been able to think of a great alternative. `storage`?